tests/Session/test_profile.py

```diff
@@ -48,5 +56,13 @@ def testListFoodStorages(self):
         self.profile.addFoodStorage(FRIDGE, fridge_name)
         self.assertEqual([freezer_name, fridge_name], self.profile.listFoodStorages(FOOD_STORAGES)) 
 
+    def testAddFoods(self):
+        # Check self.profile.addFoods returns 0 = tests that food storage is not open yet 
+        self.profile.addFoodStorage(FREEZER, freezer_name)
+        self.assertIsNotNone(self.profile.getOpenFoodStorage())
```

Comment:
You don't have to do this check again since you did it in the other test. The focus for this test is only to make sure `addFoods()` works correctly. Each unit test assumes that the other methods we wrote work correctly while we test a specific function.

Reply:
Fixed it!